juju/juju
# Display action log messages in the CLI when calling an action and also in the action result

## Description of change

Implement the facade call to watch and report on action progress messages, and wire this up to the CLI. When calling an action synchronously, the log messages are printed as they occur. When showing an action result, the log messages are included in the yaml.

## QA steps

deploy a charm with an action where the action calls action-log
juju call action

observe the log messages while the action runs

juju call action --format yaml

observe the log messages in the yaml output
